R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -1183,3 +1183,24 @@ env | map
 ```{r, echo=FALSE}
 sparkR.session.stop()
 ```
+
+```{r cleanup, include=FALSE}
+# clean up if Spark was downloaded
+# get0 not supported before R 3.2.0
+sparkDownloaded <- mget(".sparkDownloaded"[1L],
```

Comment:
Can we make this an internal util function and call it from here ?

Reply:
yes! but both call sites are outside of the package technically and we would need to call the private function with `SparkR:::`, which is kinda ugly...

I guess we could wrap the entire cleanup thing into a private/internal function (since it has to access a private flag anyway..)

Reply:
Yeah - lets do that. I might even be fine with exposing an external function 'uninstallSpark' or 'uninstallDownloadedSpark' ?

Reply:
since this needs to go into 2.2, let's not add a public method for now, we could revisit this for 2.3